### PR TITLE
python-barcode 0.15.1

### PR DESCRIPTION
**Pull request by msentissi**
python-barcode 0.15.1 :snowflake:

**Destination channel:** Snowflake

### Links

- [PKG-4751](https://anaconda.atlassian.net/browse/PKG-4751) 
- [Upstream repository](https://github.com/WhyNotHugo/python-barcode)

### Explanation of changes:

- Removed extra files from conda-forge
- Added abs.yaml
- Updated version, added testing and filled about section in recipe

[PKG-4751]: https://anaconda.atlassian.net/browse/PKG-4751?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,44 +1,55 @@
...
-  home: http://github.com/WhyNotHugo/python-barcode
+  home: https://github.com/WhyNotHugo/python-barcode
   license: MIT
   license_family: MIT
   license_file: LICENCE
-  summary: This library provides a simple way to create barcodes using only the Python standard lib
-  dev_url: http://github.com/WhyNotHugo/python-barcode
+  summary: python-barcode provides a simple way to create barcodes in Python.
+  dev_url: https://github.com/WhyNotHugo/python-barcode
+  doc_url: https://python-barcode.readthedocs.io/en/stable/
```
Comment (ryanskeith) [inline recipe/meta.yaml]:
```suggestion
  doc_url: https://python-barcode.readthedocs.io/
```
We usually drop the language encoding.

File: recipe/meta.yaml [hunk 2]
```diff
@@ -1,44 +1,55 @@
...
+    - wheel
   run:
     - python
-    - pillow
 
 test:
+  requires:
+    - pip
+    - pytest
+    - pillow
```
Comment (ryanskeith) [inline recipe/meta.yaml:35]:
```suggestion
    # pillow is optional but useful, so let's make sure we test it.
    - pillow
```

**Review by ryanskeith — APPROVED** [review]
LGTM.  Please consider incorporating my suggestions.

File: recipe/meta.yaml [hunk 3]
```diff
@@ -1,44 +1,55 @@
...
     - python-barcode = barcode.pybarcode:main
 
 
 requirements:
   host:
-    - python >=3.5
+    - python
     - pip
     - setuptools_scm
+    - wheel
```
Comment (lorepirri) [inline recipe/meta.yaml]:
```suggestion
    - setuptools >=45
    - setuptools_scm >=6.2
    - wheel
```
`setuptools` missing, plus some pinnings, see https://github.com/WhyNotHugo/python-barcode/blob/v0.15.1/pyproject.toml#L2

**Review by lorepirri — COMMENTED** [review]
A few other comments besides the ones from @ryanskeith

File: recipe/meta.yaml (hunk 3 above)
Comment (msentissi) [reply recipe/meta.yaml]: That's right I missed that, thanks !

**Review by lorepirri — APPROVED** [review]
thank you for the further changes! Let's wait for https://anaconda.atlassian.net/browse/SIR-1356 to be done before merging (close/reopen the PR to restart all of the builds first).

**Pull request merged**